src/plugins/intel_gpu/tests/unit/test_cases/hash_key_gpu_test.cpp

```diff
@@ -71,8 +71,8 @@ class check_hash_value: public ::testing::Test {
         const auto primitive_hash = primitve->hash();
         const auto params_hash = primitve->type->get_fake_aligned_params(*prim_inst->get_impl_params()).hash();
 
-        ASSERT_EQ(primitive_hash, 2197080758510296176UL);
-        ASSERT_EQ(params_hash, 4714860879383010855UL);
+        ASSERT_EQ(primitive_hash, 6924775129729406941UL);
+        ASSERT_EQ(params_hash, 8552673460001178483UL);
     }
 
```

Comment:
(random spot) could you also add some unittest for this new feature?

Reply:
added